icone_arrow_boards/README.md

```diff
@@ -0,0 +1,3 @@
+# iCone Arrow Board Testing Scripts
```

Comment:
Should we add expected environment variables in here?

Reply:
I can add descriptions. I don't want to share the password though, that should probably not be on git